docs/File-Formats.md

```diff
@@ -977,6 +973,11 @@ A structural variant data file is a tab-delimited file with one structural varia
 For an example see [datahub](https://github.com/cBioPortal/datahub/blob/master/public/msk_impact_2017/data_sv.txt). For an example see [datahub](https://github.com/cBioPortal/datahub/blob/master/public/msk_impact_2017/data_sv.txt). At a minimum `Sample_Id`, either `Site1_Hugo_Symbol`/ `Site1_Entrez_Gene_Id` or  `Site2_Hugo_Symbol`/ `Site2_Entrez_Gene_Id` and `SV_Status` are required. For the stuctural variant tab visualization (still in development) one needs to provide those field as well as `Site1_Ensembl_Transcript_Id`, `Site2_Ensembl_Transcript_Id`, `Site1_Region` and `Site2_Region`. Some of the other columns are shown at several other pages on the website. The `Class`, `Annotation` and `Event_Info` columns are shown prominently on several locations.
 **Note**: We strongly recommend all the data providers to submit genomic locations  in addition to required fields for future visualization features. 
 
+### Adding your own structural variant columns
+Additional mutation annotation columns can be added to the structural variant data file. In this way, the portal will parse and store your own structural variant fields in the database.
```

Comment:
Line breaks

```suggestion
Additional mutation annotation columns can be added to the structural variant data file. In this way, the portal will
parse and store your own structural variant fields in the database.
```

Reply:
Fixed in Fixed in [7e7ca7](https://github.com/cBioPortal/cbioportal/compare/646069c114345e4c1554f618c23396094122f4a0..7e7ca798ae5ac8bb041df89cfe1883252da1e964)